pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -2,13 +2,14 @@ package hyperconverged
 
 import (
 	"context"
+        "errors"
```

Comment:
this looks malformed, spaces instead of \t?

Reply:
I agree. I think `go fmt` should be run on this file.

Reply:
+1 go fmt it is then :-)